Assignment_3/Assignment_3_Problem_1.js

```diff
@@ -0,0 +1,37 @@
+//Assignment 3
+// Problem 1
+// Prolem statement 
+// http://www.codewars.com/kata/prefill-an-array
+// Solution
+
+function prefill(n, v) {
+
+    var flag = true;
+    var errorMessage = n + " is invalid";
+    if (typeof(n) === 'number') {
+        if (Number.isInteger(n) == false || Math.sign(n) === -1) {
+            flag = false;
+        }
+    } else if (typeof(n) === 'string') {
+        if (+n === NaN || Number.isInteger(+n) === false) {
+            flag = false;
+        }
+    } else {
+        flag = false;
```

Comment:
You can do this in another way around! put flag as false and check if the input validation is true, then change the status. else for all other cases it will be false.

Reply:
Done